meson_options.txt

```diff
@@ -0,0 +1,13 @@
+option(
+    'enable-hss',
```

Comment:
It's discouraged to make meson options start with `enable` or `disable`. I would also recommend writing more "obvious" names (e.g. sound_server and game_state instead of hss and gss)